color/src/colorspace.rs

```diff
@@ -355,6 +355,69 @@ impl ColorSpace for A98Rgb {
     }
 }
 
+/// 🌌 The Prophoto RGB color space.
```

Comment:
Should probably be ProPhoto, but clippy complains: 

```
warning: item in documentation is missing backticks
   --> color/src/colorspace.rs:358:11
    |
358 | /// 🌌 The ProPhoto RGB color space.
    |            ^^^^^^^^
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#doc_markdown
    = note: requested on the command line with `-W clippy::doc-markdown`
help: try
    |
358 | /// 🌌 The `ProPhoto` RGB color space.
    |            ~~~~~~~~~~
```

Add it to [valid idents](https://rust-lang.github.io/rust-clippy/master/index.html#doc_markdown), or slap an `#[expect(clippy::doc_markdown)]` on it?

Reply:
I believe our practice is valid idents, see for example [.clippy.toml](https://github.com/linebender/vello/blob/main/.clippy.toml) from Vello. And we should use the correct capitalization.